tools/bridge/tests/test_config.py

```diff
@@ -0,0 +1,42 @@
+import bridge.config
+
+example_logging_config = """
+[logging.root]
+level = "DEBUG"
+
+[logging.loggers."bridge.main"]
+level = "DEBUG"
+
+# web3 is too verbose with level debug
+[logging.loggers.web3]
+level = "INFO"
+
+[logging.loggers.urllib3]
+level = "INFO"
+"""
+
+
+def test_load_minimal_config(write_config, minimal_config):
```

Comment:
This test looks useless to me. It just checks that the values defined in the test itself are sensible. Am I missing something?

Reply:
If we want to have a test for config file loading, I'd suggest to start with some config, dump it, load it, and check that the result is equal. And maybe the same with some invalid values.

Reply:
> This test looks useless to me. It just checks that the values defined in the test itself are sensible. Am I missing something?

This is one of the tests that found an actual bug! I did change the `foreign_bridge.max_reorg_depth` default after writing it. 

Also, if we add a new config key without defining a default, this test will fail. I think that's useful.

If we dump a config, it will contain the full set of keys.